[docs]: Provide border to the images

shivangT opened the pull request:
-  Replaced "class" with "classname" to provide border to an image
- Update this for Introduction, Setup, Security, Tracking, Tutorials, Datasource reference

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=3853) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=3853) before we can accept your contribution.<br/><hr/>**Shivang Tiwari** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ToolJet/ToolJet?pullRequest=3853) it.</sub>

withshubh reviewed (changes requested):
- We replaced the markdown with the `<img>` tag but it is still wrapped under the `<div>` which is not required. That needs to be removed.
- The alt text is missing from all the `<img>` tags. It's good SEO - please check this @Navaneeth-pk if it's required then we will have to add it.

Navaneeth-pk commented:
@withshubh yes, alt attribute is required.

shivangT commented:
Hey @withshubh,
-The <div> containers are required when the images are not of full width. I've removed the unnecessary div but you will find it in a few places because those images need to be central aligned (removing divs makes it left aligned)
-Reverted the alt text for all <img/> tags
